<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
	border:none;
	background-color:transparent;
	background:transparent;
	padding:0;
	margin:0;
	color:#fff;
}
#centralwidget,  #mainBodyContent{
	background-color:#242424;
}

#header, #mainBody {
	background-color:#141414;
}

QPushButton{
	text-align: left;
	padding: 5px 5px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
}

#viewerBtn:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>2</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="header" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>5</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>5</number>
       </property>
       <property name="bottomMargin">
        <number>5</number>
       </property>
       <item alignment="Qt::AlignLeft">
        <widget class="QFrame" name="headerLeft">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>9</number>
          </property>
          <item alignment="Qt::AlignLeft">
           <widget class="QPushButton" name="menuBtn">
            <property name="minimumSize">
             <size>
              <width>25</width>
              <height>0</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>25</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resource/icons/menu-96.png</normaloff>resource/icons/menu-96.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="appName">
            <property name="text">
             <string>VEST</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="appNameFull">
            <property name="font">
             <font>
              <pointsize>8</pointsize>
              <bold>false</bold>
              <underline>false</underline>
              <strikeout>false</strikeout>
             </font>
            </property>
            <property name="lineWidth">
             <number>1</number>
            </property>
            <property name="text">
             <string>VEhicle Simulation Tool</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item alignment="Qt::AlignRight">
        <widget class="QFrame" name="headerRight">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="changeThemeBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resource/icons/change-theme-96.png</normaloff>resource/icons/change-theme-96.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="configSavedBtn">
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>resource/icons/settings-saved-96.png</normaloff>resource/icons/settings-saved-96.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="mainBody" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>5</number>
       </property>
       <property name="topMargin">
        <number>5</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="leftMenu" native="true">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>25</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>95</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="baseSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <property name="spacing">
              <number>0</number>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>5</number>
             </property>
             <item>
              <widget class="QFrame" name="leftMenuFrameTop">
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_5">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="viewerBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>Viewer</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/viewer-96.png</normaloff>resource/icons/viewer-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <property name="autoRepeat">
                   <bool>false</bool>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="annotationBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="text">
                   <string>Annotation</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/tag-96.png</normaloff>resource/icons/tag-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="analysisBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="text">
                   <string>Analysis</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/scatter-plot-96.png</normaloff>resource/icons/scatter-plot-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <spacer name="leftMenuVerticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QFrame" name="leftMenuFrameBottom">
               <property name="maximumSize">
                <size>
                 <width>30</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="baseSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="frameShape">
                <enum>QFrame::StyledPanel</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Raised</enum>
               </property>
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <property name="spacing">
                 <number>0</number>
                </property>
                <property name="leftMargin">
                 <number>0</number>
                </property>
                <property name="topMargin">
                 <number>0</number>
                </property>
                <property name="rightMargin">
                 <number>0</number>
                </property>
                <property name="bottomMargin">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QPushButton" name="settingBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="text">
                   <string>Setting</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/settings-96.png</normaloff>resource/icons/settings-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="helpBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="text">
                   <string>Help</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/help-96.png</normaloff>resource/icons/help-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
                <item>
                 <widget class="QPushButton" name="aboutBtn">
                  <property name="minimumSize">
                   <size>
                    <width>25</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>95</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="cursor">
                   <cursorShape>PointingHandCursor</cursorShape>
                  </property>
                  <property name="styleSheet">
                   <string notr="true">:open{
	background-color:#242424;
	border-left: 2px solid #f84119;
	font-weight: bold;
}</string>
                  </property>
                  <property name="locale">
                   <locale language="English" country="UnitedStates"/>
                  </property>
                  <property name="text">
                   <string>About</string>
                  </property>
                  <property name="icon">
                   <iconset>
                    <normaloff>resource/icons/info-96.png</normaloff>resource/icons/info-96.png</iconset>
                  </property>
                  <property name="checkable">
                   <bool>true</bool>
                  </property>
                  <property name="checked">
                   <bool>false</bool>
                  </property>
                  <property name="autoDefault">
                   <bool>false</bool>
                  </property>
                  <property name="default">
                   <bool>false</bool>
                  </property>
                  <attribute name="buttonGroup">
                   <string notr="true">leftMenuBtnGroup</string>
                  </attribute>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="mainBodyContent" native="true">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QStackedWidget" name="stackedWidget">
            <widget class="QWidget" name="viewerPage">
             <widget class="QLabel" name="label">
              <property name="geometry">
               <rect>
                <x>270</x>
                <y>180</y>
                <width>81</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>viewerPage</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="annotationPage">
             <widget class="QLabel" name="label_2">
              <property name="geometry">
               <rect>
                <x>320</x>
                <y>150</y>
                <width>131</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>annotationPage</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="analysisPage">
             <widget class="QLabel" name="label_3">
              <property name="geometry">
               <rect>
                <x>290</x>
                <y>150</y>
                <width>131</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>analyisPage</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="settingPage">
             <widget class="QLabel" name="label_4">
              <property name="geometry">
               <rect>
                <x>400</x>
                <y>190</y>
                <width>121</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>settingPage</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="helpPage">
             <widget class="QLabel" name="label_5">
              <property name="geometry">
               <rect>
                <x>260</x>
                <y>230</y>
                <width>81</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>helpPage</string>
              </property>
             </widget>
            </widget>
            <widget class="QWidget" name="aboutPage">
             <widget class="QLabel" name="label_6">
              <property name="geometry">
               <rect>
                <x>420</x>
                <y>260</y>
                <width>91</width>
                <height>16</height>
               </rect>
              </property>
              <property name="text">
               <string>aboutPage</string>
              </property>
             </widget>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="rightMenu" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="leftMenuBtnGroup"/>
 </buttongroups>
</ui>
